src/room/IReconnectPolicy.ts

```diff
@@ -0,0 +1,25 @@
+/** Controls reconnecting of the client */
+export interface IReconnectPolicy {
```

Comment:
naming nit: we don't prefix interfaces with I, keeping the same pattern.

```suggestion
export interface ReconnectPolicy {
```